src/api/app/lib/backend/connection_helper.rb

```diff
@@ -114,8 +114,13 @@ def expand_params(params, expand)
 
         expanded_params += params.delete(key).map { |value| value.to_query(key) } if params.key?(key)
       end
-      expanded_params += [params.to_query] unless params.empty?
-      expanded_params
+
+      empty_query_parameters = params.select { |_, v| v.nil? }
```

Comment:
Isn't this just [`Hash#compact`](https://ruby-doc.org/core-3.1.2/Hash.html#method-i-compact)?